tests-hivemq-platform-operator/src/integrationTest/resources/values/migration-legacy-stateful-set-values.yaml

```diff
@@ -0,0 +1,55 @@
+operator:
+  admissionWebhooks:
+    enabled: false
+
+hivemq:
+  nodeCount: 1
+  cpu: 512m
+  controllerTemplate: cluster-stateful-set.yaml
+  ports:
+    - name: "mqtt"
+      port: 1883
+      expose: true
+      patch:
+        - '[{"op":"add","path":"/spec/selector/hivemq.com~1node-offline","value":"false"},{"op":"add","path":"/metadata/annotations","value":{"service.spec.externalTrafficPolicy":"Local"}}]'
```

Comment:
I think the platform CR is missing the customization for these patches. Could be worth to add those and assert them before and after the migration.

Reply:
Good one!

Reply:
As agreed, I will create a new card for customizing services with `sessionAffinity` and assert the annotation only